### DISCUSSION TITLE
Popup - Carrot

### DISCUSSION THREAD

**Discussion by eleifer-twilio** [twilio-labs/paste#910] [Office hours] [answered]
Hi Team, 

The Explore Products page cards with Regional updates has been reviewed. In our office hours, I want to show the final design and review the components that we have decided to go with. 

One question I have outstanding is if a popover needs a carrot? I believe it does and I relayed the message to the console team but want to review with DSYS before dev starts next week.

Best,
Emma

<img width="1049" alt="Screen Shot 2020-11-04 at 2 07 40 PM" src="https://user-images.githubusercontent.com/73968607/98173299-b9c50280-1ea7-11eb-9e3c-fe7f150fee10.png">

**Comment by aayushpi** [comment]
Hi @eleifer-twilio! Thank you for the question. I see that you've signed up for office hours next week and we can talk it about it then. While there's a chance we may get to your question this week, it's likely that we'll answer it at the session. The team is taking some time off this week.

**Comment by serifluous** [comment, marked as answer]
Hey @eleifer-twilio ! The Popover component does have the beak: https://paste.twilio.design/components/popover/

Is your team maybe using the [Non-Modal Dialog primitive](https://paste.twilio.design/primitives/non-modal-dialog-primitive) instead and doing some custom styling? If so, then yep, we do recommend swapping to Popover, or adding the beak to match the Popover style.
